sprint3/controllers/articlesController.js

```diff
@@ -0,0 +1,105 @@
+import {db} from '../utils/db.js';
+import { assert } from 'superstruct';
+import { CreateArticleDto, UpdateArticleDto } from '../dtos/articles.dto.js';
+import { validatePagination } from '../utils/validatePagination.js';
+import { asyncHandler }  from '../utils/asyncHandler.js';
+
+export const getArticles = asyncHandler(async (req, res) => {
+  const { offset = 0, limit = 10, order = 'recent', search = '' } = req.query;
+  const { parsedOffset, parsedLimit } = validatePagination(offset, limit);
+
+  let orderBy;
+
+ switch (order) {
+        case 'oldst':
+          orderBy = { createdAt: 'asc' };
+          break;
+        case 'recent':
+        default:
+          orderBy = { createdAt: 'desc' };
+          break;
+        case 'updated':
+          orderBy = { updatedAt: 'desc' }; 
+          break;
```

Comment:
order를 의도하신 건지 모르겠지만, 작성해주신 코드를 보면 아래와 같이 줄여볼 수 있을 것 같아요.
```suggestion
        case 'recent':
        case 'updated':
        default:
          orderBy = { updatedAt: 'desc' }; 
          break;
```